akka-docs-dev/rst/java/code/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -55,6 +56,7 @@ public void testHostLevelExample() {
         .via(poolClientFlow)
         .runWith(Sink.<Pair<Try<HttpResponse>, Integer>>head(), materializer);
     //#host-level-example
+    assert(responseFuture != null);
```

Comment:
The disadvantage of the annotation is that it would need extra boilerplate in order to not show up in the scraped doc snippet.